organization/structure.md

```diff
@@ -5,15 +5,36 @@ This page is a work-in-progress as we define the major organizational areas of 2
 Some links might be missing or in-the-works!
 :::
 
-This section describes the major **functional areas** of 2i2c, and how they relate to each other.
+This section describes the major structure of 2i2c and how it is broken into functional groups and teams.
 Note that 2i2c is a young and small organization, so many areas have a small number (or nobody) dedicated to work within them.
 We nonetheless document these structures to understand how our work and responsibilities are divided, and to provide a roadmap for new hires and division of labor.
 
+## Functional areas
+
+Functional areas are organized around _responsibilities, skills, and career tracks_.
+Reporting lines are within a functional area, meaning that team members should report to somebody that has held a similar job in the past.
+
 % TODO: Each of these major areas should havea  `<NAME>/structure.md` page that describes that organization's structure and roles (or, a similar page with this information). We can link to each from here.
 
 - **[Organizational leadership](leadership.md)**: Steward the mission, vision, values, and major strategy of 2i2c.
 - **[Engineering](../engineering/structure.md)**: Develop and operate our technical and cloud infrastructure, to ensure that it is reliable and scalable.
+- **[Product](../product/structure.md)**: Understand the perspective of our partner communities and guide the evolution of our infrastructure and services, in order to ensure they have a positive impact.
 - **Community**: Guide and support the partner communities that we serve, to help them utilize our infrastructure in a way that aligns with 2i2c's mission.
-- **Product**: Understand the perspective of our partner communities and guide the evolution of our infrastructure and services, in order to ensure they have a positive impact.
 - **Partnerships**: Identify opportunities for impact, cultivate new relationships, and oversee our sustainability plan to ensure that we have sustainable and scalable impact.
+- **[Operations](../operations/structure.md)**: Oversees our systems for coordination, planning, and executing our work according to our goals.
 - **[Administration](../administration/css.md)**: Carry out ongoing organizational tasks that are needed to keep 2i2c running on a daily basis.
+
+## Cross-functional teams
```

Comment:
I like the distinction between functional areas and teams.
Naming might make those confusing so I appreciate the extra documentation about the idea.